src/test/regress/expected/metadata_sync_helpers.out

```diff
@@ -886,6 +886,88 @@ BEGIN TRANSACTION ISOLATION LEVEL READ COMMITTED;
 	SELECT citus_internal_update_placement_metadata(1420007, get_node_id(), get_node_id()+1);
 ERROR:  must be owner of table super_user_table
 COMMIT;
+-- the user only allowed to delete their own shards
+BEGIN TRANSACTION ISOLATION LEVEL READ COMMITTED;
+	SELECT assign_distributed_transaction_id(0, 8, '2021-07-09 15:41:55.542377+02');
+ assign_distributed_transaction_id
+---------------------------------------------------------------------
+
+(1 row)
+
+	SET application_name to 'citus';
+	\set VERBOSITY terse
+	WITH shard_data(shardid)
+		AS (VALUES (1420007))
+	SELECT citus_internal_delete_shard_metadata(shardid) FROM shard_data;
+ERROR:  must be owner of table super_user_table
+ROLLBACK;
+-- the user only allowed to delete shards in a distributed transaction
+BEGIN TRANSACTION ISOLATION LEVEL READ COMMITTED;
+	SET application_name to 'citus';
+	\set VERBOSITY terse
+	WITH shard_data(shardid)
+		AS (VALUES (1420007))
+	SELECT citus_internal_delete_shard_metadata(shardid) FROM shard_data;
+ERROR:  This is an internal function that only Citus requires to use in a distributed transaction
```

Comment:
I think I missed in a previous PR that this sentence is a bit weird. Maybe something like this:
```suggestion
ERROR:  This is an internal Citus function can only be used in a distributed transaction
```

Reply:
makes sense